### PR TITLE
translation(pt-br): getting-started-guide.md

### PR DESCRIPTION
**Pull request by LuanComputacao**
Translation to pt-br of the original en-us page

### CODE REVIEW

**Comment by nschonni** [comment]
Ping @nodejs/nodejs-pt

File: locale/pt-br/docs/guides/getting-started-guide.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+});
+
+server.listen(port, hostname, () => {
+  console.log(`Servidor rodando em http://${hostname}:${port}/`);
+});
+```
+
+Agora execute seu web server usando o comando `node app.js`. Visite o endereço `http://localhost:3000`, então você verá a mensagem "Olá mundo!".
+
+
```
Comment (nschonni) [inline locale/pt-br/docs/guides/getting-started-guide.md]: Remove extra line to make linter happy
```suggestion
```

**Comment by khaosdoctor** [comment]
Taking a look at it!

File: locale/pt-br/docs/guides/getting-started-guide.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+title: Guia: iniciando
+layout: docs.hbs
+---
+
+# Como eu começo com Node.js depois de instalá-lo?
+
+Uma ves instalado o Node.js, vamos começar nosso primeiro web server.
```
Comment (khaosdoctor) [inline locale/pt-br/docs/guides/getting-started-guide.md]:
```suggestion
Uma vez instalado o Node.js, vamos começar nosso primeiro web server.
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]
There's one small  correction

**Comment by khaosdoctor** [comment]
It's been a while since this has been opened... Is it still useful?

**Comment by XhmikosR** [comment]
Since the fork repo is gone, this is now obsolete. Feel free to make a new PR.

**Pull request closed without merging**